orangecontrib/timeseries/widgets/owlinechart.py

```diff
@@ -361,6 +364,7 @@ def __init__(self):
             # Disable bottom miniview navigator (it doesn't update)
             navigator_enabled=False, )
```

Comment:
Since this is static init configuration, passing a parameter like this here should also work:
```py
chart_legend_options_enabled=True
```